website/docusaurus.config.js

```diff
@@ -190,7 +190,7 @@ const config = {
           href: 'https://www.apache.org/',
           width: 200,
         },
-        copyright: `Apache OpenDAL (incubating) is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF. <br/><br/> Copyright © 2022-${new Date().getFullYear()}, The Apache Software Foundation Apache OpenDAL, OpenDAL, Apache, the Apache feather, and the Apache OpenDAL project logo are either registered trademarks or trademarks of the Apache Software Foundation.`,
+        copyright: `Apache OpenDAL (incubating) is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF. <br/><br/> Copyright © 2022-${new Date().getFullYear()}, The Apache Software Foundation Apache OpenDAL, OpenDAL, Apache, Apache Incubator, the Apache feather, the Apache Incubator logo and the Apache OpenDAL project logo are either registered trademarks or trademarks of the Apache Software Foundation.`,
```

Comment:
We should use:

```shell
Apache OpenDAL is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.
```

and

```shell
Copyright © 2023 The Apache Software Foundation. Apache OpenDAL, OpenDAL, and its feather logo are trademarks of The Apache Software Foundation.
```

Reply:
> Copyright © 2023

From 2023? Right.

> Apache OpenDAL, OpenDAL, and its feather logo are trademarks of The Apache Software Foundation.

Need to append `Incubator` and its logo.

Reply:
> Need to append `Incubator` and its logo.

Let's follow how kvrocks dose :rofl: